ledger/src/blockstore.rs

```diff
@@ -3602,7 +3601,7 @@ impl Blockstore {
     /// element's children slots.
     pub fn get_slots_since(&self, slots: &[Slot]) -> Result<HashMap<Slot, Vec<Slot>>> {
         let slot_metas: Result<Vec<Option<SlotMeta>>> =
-            self.meta_cf.multi_get(slots.to_vec()).into_iter().collect();
+            self.meta_cf.multi_get(slots.to_vec().into_iter()).into_iter().collect();
```

Comment:
This one is unfortunate - I'd like to update `get_slot_since()` to accept a `Vec<Slot>` instead of `&[Slot]`. Can pull that into this one or leave separately ... I initially left it out as it would modify public function in `Blockstore` / will require an update to the calling location in `ReplayStage`